src/controllers/users.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const {
+  getAll,
+  getById,
+  create,
+  remove,
+  update,
+} = require('../services/usersServices.js');
+
+const getUsers = (req, res) => {
+  const users = getAll();
+
+  res.sendStatus(200);
+  res.end(users);
+};
+
+const createUser = (req, res) => {
+  const { name } = req.body;
+
+  process.stdout.write(name);
+
+  if (!name) {
+    res.sendStatus(400);
+  }
+
+  const newUser = create(name);
+
+  res.sendStatus(201);
+  res.end(newUser);
+};
+
+const getUser = (req, res) => {
+  const { id } = req.params;
+  const foundUser = getById(id);
+
+  if (typeof id !== 'number') {

```

Comment:
it cant be number

Reply:
pls remove this check, id param will always have string type